packages/web/src/components/track/GatedContentSection.tsx

```diff
@@ -116,15 +117,17 @@ const LockedGatedContentSection = ({
   goToCollection,
   renderArtist,
   className,
-  buttonClassName
+  buttonClassName,
+  source: premiumModalSource,
+  ...other
```

Comment:
nit: are we using other? can just omit if not.

Reply:
Ah yeah I was using it to debug